### DISCUSSION TITLE
Authelia now gets stuck in a forwarding loop

### DISCUSSION THREAD

**Discussion by erbmur** [authelia/authelia#5863] [Troubleshooting] [answered]
Hey All,

I have been using authelia without any issue for a while.  And now for some reason, it gets stuck in a loop.
The page constantly refreshes itself, and I managed to catch this screenshot of a warning message that appears in the top corner.  I don't believe I changed anything in any settings.  I have also attached my settings if anybody can see anything thats wrong at a quick glance?
![image](https://github.com/authelia/authelia/assets/8417776/94f5ba41-582b-4ba7-8a35-05be687adceb)
![image](https://github.com/authelia/authelia/assets/8417776/ad7debd8-7cae-4181-bac1-954ea9bc0cc8)

**Comment by james-d-elliott** [comment]
Did you upgrade haproxy to 2.8.0? That version has a bug (lua scripts can't get the response status code) fixed in 2.8.1

**Reply by erbmur** [reply]
im running pfsense as my router, with haproxy installed in it.  This is the version i currently have installed.
![image](https://github.com/authelia/authelia/assets/8417776/3a8da3d0-c64e-4e2c-820f-9749f44e997a)

**Reply by james-d-elliott** [reply, marked as answer]
You will need to upgrade to a newer version or downgrade to an older version.

**Reply by erbmur** [reply]
Thanks for letting me know.  I will get on that ASAP

**Reply by erbmur** [reply]
That worked.  Thanks alot!
